package.json

```diff
@@ -21,7 +21,7 @@
     "shelljs": "^0.8.5"
   },
   "engines": {
-    "node": "^16.0.0 || ^18.0.0",
-    "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+    "node": "^16.11.0 || ^18.0.0",
```

Comment:
Can you remind me of why this specific version again?

Reply:
Do we need to update the `engines` in the individual packages?

Reply:
Npm 8 is only available on node 16.11.0 onwards.